src/compas/plugins.py

```diff
@@ -264,6 +281,21 @@ def wrapper(*args, **kwargs):
             # Select first matching plugin
             if selector == "first_match":
                 plugin_impl = _select_plugin(extension_point_url)
+                if plugin_impl is None:
+                    try:
+                        return func(*args, **kwargs)
+                    except PluginNotInstalledError as e:
+                        raise e
```

Comment:
Seems this one is now redundant, no? Because from now on there will always be a function to fallback to, which is the one decorated as pluggable. If this original default fails, it's either one of the following two errors.

Reply:
so the pluggable either has an implementation (which might fail for some reason), or it raises `NotImplementedError`?

Reply:
Yes I think so